src/Models/Traits/Searchable.php

```diff
@@ -38,16 +41,20 @@ public function shouldBeSearchable(): bool
         return true;
     }
 
+    public function getIndexName(): string
+    {
+        return $this->indexName ?? Str::snake(Str::pluralStudly(class_basename(static::class)));
+    }
+
     public function searchableAs(): string
     {
-        if (! config('rapidez.index')) {
+        if (! config('rapidez.store')) {
             throw new Exception('Do not use Scout directly. Please use `php artisan rapidez:index`.');
         }
```

Comment:
rapidez.store has a default value in the config. This check will never fail. We might as well remove it.
```suggestion
```

Running `STORE=2 php artisan scout:import "\Rapidez\Core\Models\Product"` actually works exactly as expected now.

Reply:
Cool, will remove then.